components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties

```diff
@@ -109,4 +111,7 @@ mandatory.claims.warning.msg.1=You need to provide consent for
 mandatory.claims.warning.msg.2=all the mandatory claims
 mandatory.claims.warning.msg.3=in order to proceed
 privacy.policy.cookies=Cookie Policy
+privacy.policy.cookies.short.description=After a successful sign in, we use a cookie in your browser to track your session. You can refer our
```

Comment:
We need to introduce new method in AuthenticationEndpointUtil. i18n(resourceBundle, key, parameters), then need to use single key. Having the multiple properties and concatenation at the code breaks the purpose of i18n

Reply:
We see your point, we have created a separate issue https://github.com/wso2/carbon-identity-framework/issues/1376 to add dynamic value concatenation in AuthenticationEndpointUtil.i18n , In this context, its necessary to break the paragraph in multiple static strings since each are rendered in separate html elements.

Reply:
+1